step-strikers-app/step-strikers-app/CodeEntryViewController.swift

```diff
@@ -0,0 +1,139 @@
+//
+//  CodeEntryViewController.swift
+//  step-strikers-app
+//
+//  Created by Alekhya Kuchimanchi on 3/1/23.
+//
+
+import UIKit
+
+class CodeEntryViewController: UIViewController {
+    
+    var popUp:UIView?
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        // Do any additional setup after loading the view.
+        assignBackground()
+        
+        // background faded shoe
+        createImage(x: 0, y: 250, w: 375, h: 375, name: "Faded Emblem")
+        
+        // giant enter code text
+        createLabel(x: 50, y: 200, w: 300, h: 200, font: "iso8", size: 45, text: "ENTER CODE:", align: .center)
+        
+        // textfield for people to enter code
+        // TODO: save this to check later
+        let textField = UITextField(frame: CGRect(x: 100, y: 350, width: 200, height: 40))
```

Comment:
Didn't want a rounded rectangle for the text field, it would not let me not include a highlight version because of force unwrapping for the buttons, and I wanted the button title text to be brown not black.